src/codeu/chat/client/commandline/Chat.java

```diff
@@ -47,11 +49,21 @@ public Chat(Context context) {
   // is willing to take another command, the function will return true. If
   // the system wants to exit, the function will return false.
   //
-  public boolean handleCommand(String line) {
+  public boolean handleCommand(String line) throws IOException {
```

Comment:
This looks like an old version of Chat. Please merge commits from the latest master into this branch and resolve any conflicts.